### DISCUSSION TITLE
Registration and Call for Content for DELPH-IN Summit

### DISCUSSION THREAD

**Discussion by fcbond** [delph-in/docs#63] [Team Posts]
G'day,

We are excited about getting ready for this year's summit!

Please register for the upcoming [DELPH-IN Summit](https://github.com/delph-in/docs/wiki/FairhavenTop) (18-22 July, Fairhaven, Washington, US) by adding your information to this wiki page if you have not already done so:

https://github.com/delph-in/docs/wiki/Fairhaven2022Participants

To add your information once you are logged in and viewing this page, click the Edit button, go to the bottom of the text page, and add your own line like the others, including your name, the dates and times you can attend, and any additional relevant notes.  Then save your changes and check to see that your addition appears as you intended.  

Please also let us know what you would like discussed at the summit, by emailing  <delphinsummit@gmail.com>.   Some possibilities are:

* site updates (five to ten minutes, including questions)
* plenary discussions (introductory problem statement by moderator) [typically up to 10+50]
* short, problem-oriented presentations (possibly backed up by shared paper pre-print) [10+20]
* long presentations: longer presentations [20+20–30]
* SIG topics
* offers or requests of tutorials

Thanks for your prompt attention!

Francis @fcbond and Luis @lmorgadodacosta

**Comment by fcbond** [comment]
G'day,

this is a reminder for everyone:

1. If you can come  to the  DELPH-IN Summit (18-22 July, Fairhaven, Washington, US) and have not yet registered, please add your information here: https://github.com/delph-in/docs/wiki/Fairhaven2022Participants
2. Woodley also reminds you to book your hotel soon
3. Please also let us know what you would like discussed at the summit **before July 1st**, by emailing delphinsummit@gmail.com. Some possibilities are:

- site updates [5-10 minutes, including questions]
- blitz (very short intro of new work or issues) [5+5]
- plenary discussions (introductory problem statement by moderator) [typically up to 10+50]
- short, problem-oriented presentations (possibly backed up by shared paper pre-print) [10+10]
- long presentations: longer presentations [20+20–30]
- SIG topics  (typically for the last two days)
- offers or requests for tutorials (typically for the last two days)

Thanks for your prompt attention!

Francis @fcbond and Luis @lmorgadodacosta

**Comment by lmorgadodacosta** [comment]
Hello everyone, 

I'm sure we are all looking forward to meet in Bellingham (or virtually) soon!

Francis and I have finalized a draft of the schedule for this year's summit, based on the emails we received. You can find the schedule here: [https://github.com/delph-in/docs/wiki/Fairhaven2022Schedule](https://github.com/delph-in/docs/wiki/Fairhaven2022Schedule)

Please take a look and make sure we didn't miss your talk, or failed to account for any reported schedule constraint.

We're also **calling for volunteers** to chair sessions and/or scribe any of the planned discussions. If you have a particular affinity with a given topic, or just want to be able to brag about your fast typing skills, drop us an email.

See you all soon,

Luis @lmorgadodacosta and Francis @fcbond

**Comment by anncopestake** [comment]
is there a link for those of us unfortunate enough not to be in person?

**Comment by emilymbender** [comment]
Woodley and Luis are working on setting that up right now. Should be posted to the Fairhaven2022Schedule page as soon as it's confirmed.

**Comment by arademaker** [comment]
I am traveling with my family in a Hotel with good internet, but eventually, with controlled time, I need to sync between family attention and the Summit. Two questions:

1. can we record the meetings? I know that people are generally against it, but what about recording for, let us say, one month only -- Zoom can make it. On YouTube you can also post an unlisted video with date/time limit. Just to allow the ones that can't make the sessions to listen to them.

2. Can I make my intro before Emily?! ;-)
